pkg/provider/gcp/secretmanager/secretsmanager_test.go

```diff
@@ -181,6 +188,128 @@ func TestSecretManagerGetSecret(t *testing.T) {
 	}
 }
 
+func TestSetSecret(t *testing.T) {
```

Comment:
Also, I see that we are not using table driven tests here. Any specific reason for that? Would be good to keep this all around.

Reply:
Table tests applied!